apps/ledger-live-desktop/src/renderer/components/TroubleshootNetworkButton.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
The troubleshooting logic looks good to me, but why still use flow and not typescript ? 🙈

Reply:
well, this is LLD. we can't do typescript unless all the "imports" are also typescript was my understanding. I cloned an existing component to make these. We will have to work on Typescript LLD as a whole.